docs/docs/typography-js.md

```diff
@@ -2,7 +2,7 @@
 title: Typography.js
 ---
 
-# Using Typography.js in Gatsby
+## Using Typography.js in Gatsby
```

Comment:
Was unsure about this; the [style guide](https://www.gatsbyjs.org/docs/gatsby-style-guide/#format-code-blocks-inline-code-and-images) says to avoid H1. I left the other headers unchanged. Happy to undo this change or decrement the other headers accordingly, as appropriate. Let me know!

Reply:
👍 the `title: Typography.js` block will be set as an h1, so bumping this to h2 is perfect.